src/components/button-toggle/button-toggle.ts

```diff
@@ -108,6 +131,17 @@ export class MdButtonToggleGroup {
     }
   }
 
+  /**
+   * Mark this group as being "touched" (for ngModel).
+   * Meant to be called by the contained button toggles upon their blur.
+   * @internal
+   */
+  touch() {
+    if (this.onTouched) {
+      this.onTouched();
```

Comment:
The extra method is quite unnecessary, because you are defining a No-Op function for the onTouched variabled, so the variable won't be undefined.

You should be able to use `onTouched` function / variable instead.